visidata/sheets.py

```diff
@@ -568,17 +567,17 @@ def drawRow(self, scr, row, rowidx, ybase, rowattr, maxheight, isNull=None):
             keysep = options.disp_keycol_sep
 
             # sepattr is the attr between cell/columns
-            sepattr = rowattr
-            sepattr = sepattr.update_attr(colors.color_column_sep, newprec=1)
+            sepcattr = rowcattr
+            sepcattr = sepcattr.update_attr(colors.color_column_sep, newprec=1)
```

Comment:
This can just be `sepcattr = rowcattr.update_attr(colors.color_column_sep, newprec=1)`